pkgs/by-name/qr/qrtool/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "qrtool";
+  version = "0.8.4";
+
+  src = fetchFromGitHub {
+    owner = "sorairolake";
+    repo = "qrtool";
+    rev = "v${version}";
+    sha256 = "sha256-FoWUGhNfVILpYxmsnSzRIM1+R9/xFxCF7W1sdiHaAiA=";
+  };
+
+  cargoSha256 = "sha256-mtejnHCkN2krgFAneyyBpvbv5PZO3GigM2DJqrbHim4=";
+
+  meta = with lib; {
```

Comment:
Could you add the changelog here? It looks like it's at `changelog = "https://github.com/sorairolake/qrtool/blob/v{version}/CHANGELOG.adoc";` but do double-check that. thanks!

Reply:
Done, thanks.